src/main/resources/languages/messages_en.yml

```diff
@@ -42,6 +43,12 @@ guide:
 
   cheat:
     no-multiblocks: '&4You cannot cheat in Multiblocks, you have to build them!'
+
+  backpack:
+    invalid-id: '&4The id must be a non-negative integer!'
+    player-never-joined: '&4No player with that name has ever joined the server!'
+    backpack-does-not-exist: '&4That backpack does not exist!'
+    restored-backpack-given: 'Backpack restored successfully! Added to your inventory!'
```

Comment:
Why exactly is this in the "guide" section? Doesn't seem to have anything to do with the Slimefun guide to me.
Should probably rather be in the "commands" section.